README.md

```diff
@@ -113,7 +114,7 @@ module.exports = {
 {
   "scripts": {
     "test": "npm run lint",
-    "lint": "eslint .",
+    "lint": "eslint --ext .js,.ts,.tsx .",
```

Comment:
it's too bad you can't set this with a config :( https://github.com/eslint/eslint/issues/10828

Reply:
Agree :cry: